conda/plugins/manager.py

```diff
@@ -50,19 +54,38 @@ def load_plugins(self, *plugins) -> list[str]:
                 plugin_names.append(plugin_name)
         return plugin_names
 
-    def load_setuptools_entrypoints(self, *args, **kwargs) -> int:
-        """
-        Overloading the parent method from pluggy to add conda specific exceptions.
-
-        See :meth:`pluggy.PluginManager.load_setuptools_entrypoints` for
-        more information.
+    def load_entrypoints(
+        self, group: str, name: str | None = None
+    ) -> int:
+        """Load modules from querying the specified setuptools ``group``.
+        :param str group: Entry point group to load plugins.
+        :param str name: If given, loads only plugins with the given ``name``.
+        :rtype: int
+        :return: The number of plugins loaded by this call.
         """
-        try:
-            return super().load_setuptools_entrypoints(*args, **kwargs)
-        except Exception as err:
-            raise PluginError(
-                f"Error while loading conda plugins from entrypoints: {err}"
-            )
+        count = 0
+        for dist in list(distributions()):
```

Comment:
Is this the retconned "same thing setuptools did" equivalent function?

Reply:
Yep, importlib.metadata is the pkg_resources equivalent